content/en/docs/tasks/run-application/run-single-instance-stateful-application.md

```diff
@@ -49,14 +49,19 @@ Note: The password is defined in the config yaml, and this is insecure. See
 for a secure solution.
 
 {{< code file="mysql-deployment.yaml" >}}
+{{< code file="mysql-pv.yaml" >}}
+
+1. Deploy the PV and PVC of the YAML file:
+
+        kubectl create -f https://k8s.io/docs/tasks/run-application/mysql-pv.yaml
```

Comment:
can you verify if this command actually works? there were complaints that urls cannot be used as the '-f' parameter value.

Reply:
@tengqm `https://k8s.io/docs/tasks/run-application/mysql-pv.yaml` does not exist, I created it in the PR according to your suggestion, I use my github file `https://raw.githubusercontent.com/chenhonggc/kubernetes.github.io/format_style/content/en/docs/tasks/run-application/mysql-pv.yaml` to verify that this command is fine.

The cluster: `https://www.katacoda.com/courses/kubernetes/playground`
The output:
```
master $ kubectl version
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3
465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:55:54Z", GoVersion:"go1.9.3", Compiler:
"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3
465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:
"gc", Platform:"linux/amd64"}

master $ kubectl create -f https://raw.githubusercontent.com/chenhonggc/kubernetes.github.io/format_style/c
ent/en/docs/tasks/run-application/mysql-pv.yaml
persistentvolume "mysql-pv-volume" created
persistentvolumeclaim "mysql-pv-claim" created

master $ kubectl get pv,pvc
NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                    STORAGECLASS   REASON    AGE
mysql-pv-volume   20Gi       RWO            Retain           Bound     default/mysql-pv-claim   manual              27s

NAME             STATUS    VOLUME            CAPACITY   ACCESS MODES   STORAGECLASS   AGE
mysql-pv-claim   Bound     mysql-pv-volume   20Gi       RWO            manual         27s 
```

Please review again. Thanks!

Reply:
Thanks for the confirmation.